controllers/install/lookout_controller.go

```diff
@@ -298,6 +273,66 @@ func createLookoutMigrationJob(lookout *installv1alpha1.Lookout) *batchv1.Job {
 			},
 		},
 	}
+	if lookout.Spec.Resources != nil {
+		deployment.Spec.Template.Spec.Containers[0].Resources = *lookout.Spec.Resources
+	}
+	return &deployment
+}
 
-	return &job
+func createLookoutIngressWeb(lookout *installv1alpha1.Lookout) *networking.Ingress {
```

Comment:
I need to re-match this with the actual IngressWeb Template.

I am assuming the other templates match well enough to get the e2e tests passing?

Reply:
I have a bug in binoculars that spams the logs but the tests pass.  Something to do with ingress rules.  

No one of us have a fully working deployment currently.  Maybe Dejan but he doesn't have ingress or other items there so we still gotta fully work that out.

Reply:
Good to know. I will get the tests passing and we can fix these problems down the line.